api/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha1 contains API Schema definitions for the gcp v1alpha1 API group
+//+kubebuilder:object:generate=true
+//+groupName=gcp.managed.openshift.io
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: "gcp.managed.openshift.io", Version: "v1alpha1"}
```

Comment:
TODO: we might want to move this to `v1` as the schema didn't change in at least a year and we are using it in prod for a long while

Reply:
This would be another major change. Can we keep that for another PR if possible?